TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java

```diff
@@ -385,9 +385,13 @@ public void run() {
                     mSnackbarLoadNewMap.setAction(getString(R.string.snackbar_action_text_load_new_map), new View.OnClickListener() {
                         @Override
                         public void onClick(View view) {
-                            mParameterNode.changeSettingsToLocalizeInMap(mapUuid, getString(R.string.pref_create_new_map_key),
-                                    getString(R.string.pref_localization_mode_key), getString(R.string.pref_localization_map_uuid_key));
-                            restartTango();
+                            try {
```

Comment:
I think you also need to add a try/catch block here: https://github.com/Intermodalics/tango_ros/pull/359/files#diff-a79d89dbb3540e8c2b65c1f9155800a7R344